README.md

```diff
@@ -162,6 +170,56 @@ It is also a really common case to communicate with some external API. But in te
 ```
 From this moment, first `getOutsideApiResponse` call will receive `array('WithMessage')` and any further call will cause an exception. To make maintaining of external API responses as easy as possible ApiTestCase provides a method to load data directly from json file `$this->getJsonResponseFixture('mocked_response')`. `mocked_response.json` file should be placed in a src/YourBundle/Tests/Responses/Mocked/ folder, or any other defined in phpunit.xml file.
 
+Test with database fixtures
+===========================
+Api test case is integrated with [Alice](https://github.com/nelmio/alice) thanks to this you can easily load your fixtures when you need them. You have to define your fixtures and place it in proper directory.
+
+Here is some example how to define your fixtures and usage case. For more information how to define your fixtures check documentation of ``nelmio/alice``.
+
+Let's start with defining resource to store.
+
+```php
+    class Product
+    {
+        private $id;
+        private $name;
+        private $price;
+    
+        // Proper setters and getters.
+    }
+```
+
+Now we are almost ready to go, one thing we need to do is define fixtures.
+
+```yml
+    Lakion\ApiTestCase\Test\Entity\Product:
+        product1:
+            name: 'Phone'
+            price: 200
+        product2:
+            name: 'Book'
+            price: 15
+        product3:
+            name: 'Mug'
+            price: 5
+```
+
+Ok so we got our data now let's proceed with our testing case.
+
+Assume that you got indexAction in your ProductController to test it you need to load your defined fixtures. There is following method to achieve this ``->loadPlatformData()``.
```

Comment:
For example: `In order to test your index action, you will need to load your defined fixtures.`